pg2kinesis/formatter.py

```diff
@@ -104,7 +105,7 @@ def _preprocess_wal2json_change(self, change):
 
         change_dictionary = json.loads(change)
         if not change_dictionary:
-            return None
+            return []
```

Comment:
Why this change?

Reply:
this is used like this:
```
        pp_changes = self._preprocess_wal2json_change(change)
        return [self.produce_formatted_message(pp_change) for pp_change in pp_changes]
```

if you return None, this will end up raising an exception, whereas if there are no changes to process, returning an empty list will be a no-op and wait for the next change in the repslot.